packages/flutter/test/material/icons_test.dart

```diff
@@ -122,6 +122,27 @@ void main() {
 
     await expectLater(find.byType(Wrap), matchesGoldenFile('test.icons.sample3.png'));
   }, skip: isBrowser); // https://github.com/flutter/flutter/issues/39998
+
+  // Regression test for https://github.com/flutter/flutter/issues/103202.
+  testWidgets('Another sample of icons look as expected', (WidgetTester tester) async {
+    await _loadIconFont();
+
+    await tester.pumpWidget(MaterialApp(
+      home: IconTheme(
+        data: const IconThemeData(size: 200),
+        child: Wrap(
+          children: const <Icon>[
+            Icon(Icons.repeat_on),
```

Comment:
Are these just a sampling of the new icons or were the chosen for a specific purpose? Should we have a test that uses all the new ones? Or is that too onerous?

Reply:
These are the ones that had an issue associated with them. There are too many icons to test so I usually just add a sample.